src/main/java/edu/ie3/datamodel/utils/validation/SystemParticipantValidationUtils.java

```diff
@@ -288,7 +288,13 @@ private static Try<Void, InvalidEntityException> checkEvType(EvTypeInput evTypeI
     return Try.ofVoid(
         () ->
             detectZeroOrNegativeQuantities(
-                new Quantity<?>[] {evTypeInput.geteStorage(), evTypeInput.geteCons()}, evTypeInput),
+                new Quantity<?>[] {
+                  evTypeInput.geteStorage(),
+                  evTypeInput.geteCons(),
+                  evTypeInput.getsRatedAC(),
+                  evTypeInput.getsRatedDC()
+                },
+                evTypeInput),
         InvalidEntityException.class);
```

Comment:
This is just some validation method for battery capacity and ev consumption. The rated power should not be in there.

Reply:
@danielfeismann I think we should check the rated DC power here. AC is checked in ``SystemParticipantValidationUtils.checkType``. But there is no test for DC yet.